test/performance/README.md

```diff
@@ -15,32 +15,81 @@ different rate, you can write your own pacer by implementing
 interface. Custom pacer implementations used in Knative tests are under
 [pacers](https://github.com/knative/pkg/tree/main/test/vegeta/pacers).
 
-## Benchmarking using Mako
 
-The benchmarks were originally built to use [mako](https://github.com/google/mako), but currently
-running without connecting to the Mako backend, and collecting the data using
-a Mako sidecar stub.
+## Testing architecture
 
-### Run without Mako
+The performance tests are based on Kubernetes Jobs running Golang code based on different [benchmarks](#benchmarks).
+The script [performance-tests.sh](./performance-tests.sh) first creates a cluster in GKE, installs Serving with specific settings
```

Comment:
I guess the file path for `performance-tests.sh` is not right as the file [test/performance](https://github.com/knative/serving/pull/14289/files#diff-bea0df81e6ece5471ea1adbd1dc1f112d3aa31b7a448288d96a270a16620d909) has been deleted.

Reply:
I'd like to drop `mako` as we no longer use it, but for now it is necessary in the script because this is the CI jobs target. Shortly before we merge this PR here, I'll change the test-targets on CI as well. So the name in the readme will be correct.

Reply:
https://github.com/knative/infra/pull/182